src/linuxforhealth/csvtofhir/fhirrs/procedure.py

```diff
@@ -31,7 +31,7 @@ def get_procedure_sequence_extensions(incoming_data: ProcedureCsv) -> Union[List
         return None
     extensions_list: List[Extension] = []
     ext: Extension = Extension.construct(url=ExtensionUrl.PROCEDURE_SEQUENCE_EXTENSION_URL)
-    ext.valueString = incoming_data.procedureEncounterSequenceId
+    ext.valuePositiveInt = incoming_data.procedureEncounterSequenceId
```

Comment:
do we need error checking for it its not an int?  like in the original example it was 1A?

Reply:
Not sure where 1A came from, I had checked our data sets and they were all numeric.